### DISCUSSION TITLE
FR: chat-like page for ChatGPT interaction

### DISCUSSION THREAD

**Discussion by dv0zn3r** [briansunter/logseq-plugin-gpt3-openai#186] [Ideas]
Would be useful to have a chat-like page for interaction with ChatGPT in Logseq. Something like that would be possible, if Logseq had a shortcut for jumping to the last block. See feature request [here](https://discuss.logseq.com/t/add-shortcut-to-go-to-the-last-block-on-the-page/2917). However, perhaps a more elegant solution may be developed. In any event, this might be one of the issues you would want to discuss in [cookbook](https://github.com/briansunter/logseq-plugin-gpt3-openai/issues/12) in re organising ChatGPT answers.

**Comment by wirtzdan** [comment]
I was thinking about this too the other day. I think it would be super helpful, but probably also quite some work to deliver. I wonder if Logseq would be open to sponsoring this development, similar to how they made the whiteboard feature happen.

I would think, that the natural place for such a chat interface, could be in the right sidebar. That way, you could reference your main page, or work and then use the chat, to get help from the AI.  

Here a quick sketch of where I think the placement would be nice: 
![CleanShot 2023-03-08 at 10 43 44](https://user-images.githubusercontent.com/18514829/223678658-b109baf3-ec35-4a9c-8713-ede663a9e404.png)

**Comment by briansunter** [comment]
I really like this idea. I'll think about adding it in future versions.

**Comment by dv0zn3r** [comment]
fyi [new ChatGPT plugin](https://twitter.com/debanjandhar14/status/1638567560760426496?s=46&t=AlZ4R3WubQStzKZa18_k4w) coming soon with such feature
